src/OpenLoco/src/Audio/Audio.cpp

```diff
@@ -184,6 +184,16 @@ namespace OpenLoco::Audio
     {
         disposeDSound();
         initialiseDSound();
+
+        // Resume title music if it should be playing
+        if (isTitleMode())
+        {
+            auto& cfg = Config::get();
+            if (cfg.audio.playTitleMusic)
+            {
+                Audio::playMusic(Environment::PathId::css5, Config::get().old.volume, true);
+            }
+        }
```

Comment:
This `reinitialise` function is only called from `Audio::setDevice`, which is only called by the option window. I'd prefer we handle the music logic there instead.

Reply:
I did it this way because making sure that the music is in the correct state felt more like a 'reinitialise audio' thing than a 'set audio device' thing to me, but I'm happy to change it if no one agrees or I'm outvoted

Reply:
Yeah, I can see why you did it this way. Ideally, the Audio subsystem doesn't have to deal with game-specific logic. That way, it could be easily re-used by other projects in the future. @ZehMatt Do you agree?